lab3/graph.py

```diff
@@ -0,0 +1,38 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+omega = 0.5
+    
+data = np.loadtxt('points.txt')
+X = np.array([x[0] for x in data])
```

Comment:
why? x[0] is already a numpy array data (slice view to be more specific). So why do you have to make explicit copy here? BTW if you do some numpy examples review - you'll se you even can read composite dataset and use it in a way data['x'] , data['t'], etc. The same behavior is conventional and implemented by pandas dataframes as well.

Reply:
I understand that I could write it all in one line, but then I would have to search the Internet for the use of pandas, since I know about its existence, but I don't remember how to use it, so it was more convenient for me to write 5 lines at once than to spend time in Google and write one.